front/src/applications/operationalStudies/consts.ts

```diff
@@ -180,7 +180,7 @@ export interface OsrdConfState {
   departureTime: string;
   destination?: PointOnMap;
   vias: PointOnMap[];
-  suggeredVias: Path['steps'];
+  suggeredVias: Path['steps'] | PointOnMap[];
```

Comment:
Why did you add this ? Is it for the tests ?
```suggestion
  suggeredVias: Path['steps'] | PointOnMap[];
```

Reply:
because I saw that in `adjustConfWithTrainToModify.ts` file, when we have `dispatch(updateSuggeredVias(compact(path.steps.map((via) => convertStepToPointOnMap(via)))));` we pass an array of type PointOnMap[] in updateSuggeredVias...

Reply:
No `convertStepToPointOnMap` returns `PointOnMap`